modules/doc/content/getting_started/examples_and_tutorials/tutorial04_meshing/step11_rgmb_het_hom.md

```diff
@@ -0,0 +1,128 @@
+
+- +To define single assemblies directly with PinMeshGenerators for stitching with [CoreMeshGenerator.md]+, [PinMeshGenerator.md] is used with [!param](/Mesh/PinMeshGenerator/use_as_assembly) set to `true`.
+- In addition, [!param](/Mesh/PinMeshGenerator/homogenized) = `true` is used to indicate that this region is homogenized and [SimpleHexagonGenerator.md] should be called to discretize the assembly instead of [PolygonConcentricCircleMeshGenerator.md].
+
+!listing reactor_examples/rgmb_abtr/rgmb_abtr_het_mesh.i
+         id=tutorial04-rgmb_abtr-hom-assembly
+         caption=ABTR RGMB homogeneous assembly example.
+         block=Mesh/reflector_assembly
+
+Tips
+
+- For homogenized assemblies, each assembly has only 1 radial region, so each column in `region_ids` pertains to the region ID of each axial level of the homogenized assembly.
+- While the mesh is still 2D during this step, the axially dependent region IDs are stored for later use during the extrusion step.
+
+## Heterogeneous core using CoreMeshGenerator
+
+Now that all heterogeneous and homogeneous assemblies have been defined, they are placed into a lattice using [CoreMeshGenerator.md]. While [CoreMeshGenerator.md] still requires a perfect hexagonal pattern like [PatternedHexMeshGenerator.md], it automatically handles dummy assembly creation and deletion. The user need only provide a fake mesh input reference `dummy` (this object has not been actually created) and tell [CoreMeshGenerator.md] through the [!param](/Mesh/CoreMeshGenerator/dummy_assembly_name) parameter that the mesh input called `dummy` is a dummy assembly (empty space). The dummy assemblies will be created and deleted behind the scenes with no effort from the user.
+
+- Since we want to extrude the 2D core, we use the [!param](/Mesh/CoreMeshGenerator/extrude)=`true` parameter within [CoreMeshGenerator.md]. The step simultaneously extrudes the geometry and applies the regions IDs to all axial layers as defined in the [PinMeshGenerator.md] objects.
+- Behind the scenes, extra element IDs `assembly_id` and `plane_id` are automatically applied.
+- Since [!param](/Mesh/ReactorMeshParams/flexible_assembly_stitching) = `true` in [ReactorMeshParams.md], the different assembly types are stitched together without hanging nodes.
+
+!listing reactor_examples/rgmb_abtr/rgmb_abtr_het_mesh.i
+         id=tutorial04-rgmb_abtr-het_core
+         caption=ABTR RGMB core example.
+         block=Mesh/het_core
+
+!alert tip
+[!param](/Mesh/CoreMeshGenerator/extrude) = `true` indicates extrusion should happen at this step (this should only be set to true once in the entire input file)
+
+!media tutorial04_meshing/rgmb_abtr_het_core_eeid.png
+       id=tutorial04-rgmb_abtr_het_core
+       caption=RGMB-generated 3D heterogeneous core mesh, colored by various IDs defined on the mesh.
+       style=width:75%;display:block;margin-left:auto;margin-right:auto;
+
+## Equivalent homogeneous core using Griffin's EquivalentCoreMeshGenerator
```

Comment:
or should the tutorial end right here for MOOSE-only?

Reply:
(the griffin content can stay for convenience)

Reply:
Added a description here to indicate end of MOOSE content and beginning of Griffin content